templates/partials/modals/cleanup.html

```
@@ -1,5 +1,5 @@
 <!-- Cleanup Modal -->
-<div id="cleanup-modal" class="modal" role="dialog" aria-label="Clean Up Folders" aria-modal="true">
+<div id="cleanup-modal" class="modal" role="dialog" aria-label="Clean Up Folders" aria-modal="true" aria-hidden="true">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)